trestle/core/models/elements.py

```diff
@@ -416,3 +416,9 @@ def __eq__(self, other: object) -> bool:
             return False
 
         return self.get() == other.get()
+
+
+def get_singular_model_from_json(json_path: str, model_obj: OscalBaseModel):
+    """Get the singular model type represented by the full path with wildcards."""
+    singular_model = utils.get_target_model(json_path.split('.'), model_obj)
+    return singular_model
```

Comment:
For the purpose of not splitting at the level of strings I just want to know if the string specifying the split elements, including wildcards, is too fine or not.  It gets involved because the string is in the context of the directory structure.  The code for parsing the string in the context of the directory is complicated and I would like it cleaned up.